multicluster/controllers/multicluster/commonarea/clusterinfo_importer_test.go

```diff
@@ -149,26 +149,30 @@ func TestResourceImportReconciler_handleClusterInfo(t *testing.T) {
 	}
 
 	tests := []struct {
-		name                string
-		existingCIResImport *mcsv1alpha1.ResourceImport
-		existingCIImport    *mcsv1alpha1.ClusterInfoImport
-		expectedCIImport    *mcsv1alpha1.ClusterInfoImport
-		isDelete            bool
+		name                  string
+		existingCIResImport   *mcsv1alpha1.ResourceImport
+		existingCIImport      *mcsv1alpha1.ClusterInfoImport
+		expectedCIImport      *mcsv1alpha1.ClusterInfoImport
+		isDelete              bool
+		expNumInstalledResImp int
```

Comment:
Maybe name it as `expectedInstalledResImpSize`

Reply:
Done